spec/models/plan_spec.rb

```diff
@@ -9,4 +9,17 @@
     expect(plan.price).to eq(15.50)
     expect(plan.token).not_to be_nil
   end
+
+  it 'saves price history' do
+    customer = Fabricate(:customer)
+    plan = Plan.create!(name: 'Plano Bronze', price: 15.50, customer: customer)
+
+    expect(plan.plan_price_history.last.price).to eq(15.50)
+
+    plan.price = 20.99
+
+    plan.save!
+
+    expect(plan.plan_price_history.last.price).to eq(20.99)
```

Comment:
```suggestion
    plan = Plan.create!(name: 'Plano Bronze', price: 15.50, customer: customer)
    plan.update!(price: 20.99)

    expect(plan.plan_price_history.pluck(:price)).to eq([15.50, 20.99])
```

Aqui você pode trazer os dois valores de uma vez e já verificar o que acha?